sanssouci/reference_families.py

```diff
@@ -87,3 +87,94 @@ def inverse_beta_template(y, k, m):
 
     """
     return beta.cdf(y, k, m + 1 - k)
+
+
+def regularised_inverse_beta_template(y, k, m, reg=100):
```

Comment:
You don't want these functions to be called directly by end users, do you ? If not call it 
`_regularised_inverse_beta_template`
(same with next functions)